Tasks/AzureWebAppV1/package.json

```diff
@@ -20,8 +20,11 @@
     "@types/mocha": "^5.2.7",
     "@types/node": "^10.17.0",
     "@types/q": "1.0.7",
-    "azure-pipelines-tasks-azurermdeploycommon": "^3.221.1",
+    "azure-devops-node-api": "11.2.0",
+    "azure-pipelines-task-lib": "^4.1.0",
```

Comment:
Since this is a Node10 task, you should use `3.4.0` for "azure-pipelines-task-lib"

Reply:
Done.